mcstatus/pinger.py

```diff
@@ -1,34 +1,40 @@
 import datetime
 import json
 import random
-from typing import List, Optional
-
-from mcstatus.protocol.connection import Connection
-
-COLOR_MAP = {
-    "dark_red": "4",
-    "red": "c",
-    "gold": "6",
-    "yellow": "e",
-    "dark_green": "2",
-    "green": "a",
-    "aqua": "b",
-    "dark_aqua": "3",
-    "dark_blue": "1",
-    "blue": "9",
-    "light_purple": "d",
-    "dark_purple": "5",
-    "white": "f",
-    "gray": "7",
-    "dark_gray": "8",
-    "black": "0",
+from typing import List, Optional, Union
+
+from mcstatus.protocol.connection import Connection, TCPSocketConnection, TCPAsyncSocketConnection
+
+STYLE_MAP = {
+    "bold": "l",
+    "italic": "o",
+    "underlined": "n",
+    "obfuscated": "k",
```

Comment:
These could be a bit complicated. §m and §r are missing.
https://minecraft.fandom.com/wiki/Formatting_codes